### DISCUSSION TITLE
How to embed other posts into the main post?

### DISCUSSION THREAD

**Discussion by heydc7** [adityatelange/hugo-PaperMod#1317] [Q&A]
I want to create a page where I can list all other sub-pages like this:

![2023-10-09 03_17_49-HackTheBox _ Aditya Telange - Brave](https://github.com/adityatelange/hugo-PaperMod/assets/39642060/3f1d6c70-ee21-4319-aa73-d49dcbb32bdb)

How to implement it?

**Comment by adityatelange** [comment]
Create a file `_index.md` inside you're section suppose `writeups/` where we can have other posts in the section shown as list.

**Comment by heydc7** [comment]
Can you share the sample format for content in `_index.md`?

**Reply by jasonbraganza** [reply]
A blank _index.md should do fine
